include/CppUTest/JUnitTestOutput.h

```diff
@@ -74,6 +74,9 @@ class JUnitTestOutput: public TestOutput
     virtual void writeFailure(JUnitTestCaseResultNode* node);
     virtual void writeFileEnding();
 
+private:
+    ConsoleTestOutput* currentConsoleOutput_;
+    ConsoleTestOutput defaultConsoleOutput_;
```

Comment:
Without having an actual case for using it, I'd probably not want to add the complexity. This was also why I eventually simplified the CommandLineTestRunner